File: Codewars/VowelCount.py
# Return the number (count) of vowels in the given string.

def get_count(sentence):
    valid_vows = "aeiouAEIOU"
    vows_found = 0
    for poss_vow in sentence:
        if poss_vow in valid_vows:
            vows_found += 1
            
    return vows_found
        

File: Codewars/VowelCount.ts
export class Kata {
    static getCount(str: string): number {
      let vowels: string = "aeiouAEIOU";
      let count: number = 0;
      for (let vowel = 0; vowel < str.length; vowel++) {
        if (vowels.includes(str[vowel])) {
          count++;
        }
      }
      return count
    }
  }